e2e/specs/table.spec.ts

```diff
@@ -37,40 +37,72 @@ test.beforeEach(async ({ page }) => {
-  await el.click();
+  await page.goto('/subject-matter/learning-resource/new');
+  await page.locator('[data-cy="slate-editor"]').click();
   await page.locator('[data-cy="slate-block-picker"]').click();
-  await expect(page.locator('[data-cy="slate-block-picker-menu"]')).toBeVisible();
   await page.locator('[data-cy="create-table"]').click();
-  const caption = page.locator('caption');
-  await caption.click();
+});
+
+test('all table functions work', async ({ page }) => {
+  expect(await page.locator('th').count()).toEqual(2);
+  expect(await page.locator('td').count()).toEqual(2);
+  expect(await page.locator('tr').count()).toEqual(2);
+  await page.locator('caption').click();
   await page.keyboard.type('TITTEL!');
   await expect(page.locator('caption')).toHaveText('TITTEL!');
   await page.locator('tbody > tr > td').first().fill('Cell');
-  const header = page.locator('thead > tr > th').first();
-  await header.click();
+  await page.locator('thead > tr > th').first().click();
   await page.keyboard.type('Header 1');
   await page.keyboard.press('ArrowRight');
   await page.keyboard.type('Header 2');
+  await expect(page.locator('thead > tr > th').first()).toHaveText('Header 1');
+  await expect(page.locator('thead > tr > th').last()).toHaveText('Header 2');
   await page.locator('[data-cy="column-add"]').click();
+  expect(await page.locator('th').count()).toEqual(3);
+  expect(await page.locator('td').count()).toEqual(3);
+
   await page.locator('thead > tr > th').last().click();
   await page.keyboard.type('Test new column');
+
   await page.locator('[data-cy="row-remove"]').click();
+  expect(await page.locator('th').count()).toEqual(0);
+  expect(await page.locator('td').count()).toEqual(3);
+  expect(await page.locator('tr').count()).toEqual(1);
   await page.locator('[data-cy="head-add"]').click();
+  expect(await page.locator('tr').count()).toEqual(2);
+  expect(await page.locator('th').count()).toEqual(3);
+  expect(await page.locator('td').count()).toEqual(3);
+
   await page.locator('thead > tr > th').last().click();
   await page.keyboard.type('Test new header');
   await page.keyboard.press('ArrowDown');
   await page.locator('[data-cy="row-add"]').click();
   await page.locator('tbody > tr > td').last().click();
+  expect(await page.locator('tr').count()).toEqual(3);
+  expect(await page.locator('td').count()).toEqual(6);
   await page.keyboard.type('Test new row');
   await page.locator('[data-cy="toggle-row-headers"]').click();
+  expect(await page.locator('th').count()).toEqual(5);
+  expect(await page.locator('td').count()).toEqual(4);
   await page.locator('[data-cy="column-remove"]').click();
+  expect(await page.locator('th').count()).toEqual(4);
+  expect(await page.locator('td').count()).toEqual(2);
   await page.locator('[data-cy="row-remove"]').click();
+  expect(await page.locator('tr').count()).toEqual(2);
   await page.locator('[data-cy="table-remove"]').click();
 });
```

Comment:
Jeg har lyst til å splitte denne opp til flere tester (en eller to), det er ikke strengt tatt nødvendig men kan gjør det lettere å finne ut hva som er feil da denne tester hele tabellen